pkg/client/create_builder.go

```diff
@@ -201,7 +208,7 @@ func (c *Client) createBaseBuilder(ctx context.Context, opts CreateBuilderOption
 	return bldr, nil
 }
 
-func (c *Client) fetchLifecycle(ctx context.Context, config pubbldr.LifecycleConfig, relativeBaseDir, os string, architecture string) (builder.Lifecycle, error) {
+func (c *Client) fetchLifecycle(ctx context.Context, config pubbldr.LifecycleConfig, relativeBaseDir, o string, architecture string) (builder.Lifecycle, error) {
```

Comment:
Do no change the `os` variable name, it is more descriptive that `o`

Reply:
Changed because I needed to use a function from the os package. Will try an alternative way